### ISSUE TITLE
Couldn't connect to accessibility bus

### ISSUE THREAD

**Issue by wwat** [NixOS/nixpkgs#6082]
As root and as a normal user (logged in via ssh which can both run xclock). All I wanted to do was install a VM (because NixOS hasn't packaged something), but apparently running a VM is already too difficult.
```
** (virt-manager:19159): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-4nWd4NTszZ: Connection refused
```

This results in a message saying that no virtualization system (e.g. kvm) is detected, which is wrong because
```
 lsmod | grep kvm 
```
returns 
```
kvm_intel             131623  0 
kvm                   396575  1 kvm_intel
```

**Comment by bjornfor** [comment]
Need more info. I'm running libvirt VMs in NixOS without problem here.

**Comment by wwat** [comment]
The application `virt-manager` returns this response. How can it be my responsibility to figure out what its problem is? Apparently, `virt-manager` took on a dependency without adding the proper error handling to inform the user about exactly what is wrong. In this case, certainly when running as root, it has full capabilities to inspect the complete system to determine what exactly is wrong, but it doesn't do that. 

I am not sure what more info I can give. I was actually also able to run a libvirt VM, but I do get "could not detect a default hypervisor" when I start `virt-manager`. 

I have
```
  services.dbus.enable = true;
```
so, if that is set, and I install virt-manager, I expect  that under *all* circumstances, the configuration works, or the configuration gives me an error message and thus it should fail to apply the configuration. This is not what is happening; the set of requirements for `virt-manager` to work is not complete, so it's wrong. As to what's wrong? I have no idea. Whoever designed `virt-manager` should know that and that's not me.

**Comment by bjornfor** [comment]
Ah, I remember that "could not detect a default hypervisor" message. I think I also saw it on an Ubuntu system I tested. I wonder if virt-manager is supposed to get something from avahi...? Anyway, I added a QEMU/KVM type connection (once) and virt-manager never complained again.

**Comment by alexhenrie** [comment]
I'm having the same problem here. When I try to run virt-manager, I get:

> ** (virt-manager:4308): WARNING **: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.

And then the GUI displays the message:

> Could not detect a default hypervisor. Make sure the appropriate virtualization packages are installed (kvm, qemu, libvirt, etc.), and that libvirtd is running.
>
> A hypervisor connection can be manually added via File->Add Connection

I am using Arch Linux, and I installed virt-manager with:

> sudo pacman -S virt-manager dmidecode ebtables qemu
> sudo systemctl enable libvirtd
> sudo systemctl start libvirtd

Is there some trick to virt-manager that I don't understand?
